src/components/App.css

```diff
@@ -0,0 +1,52 @@
+.App {
+    display: flex;
+    flex-direction: column;
+    width: 325px;
+    height: auto;
+    margin: 30px auto;
+    padding: 20px 20px 9px;
+    background: #9dd2ea;
+    background: linear-gradient(#37b7f1, #07abf7);
+    border-radius: 3px;
+    box-shadow: 0px 4px #056e9e, 0px 10px 15px rgba(0, 0, 0, 0.2);
+  }
+
+  .display {
+     height: 40px;
+     width: 303px;
+     float: right;
+     padding: 0 10px;
+     background: rgba(0, 0, 0, 0.2);
+     border-radius: 3px;
+     box-shadow: inset 0px 4px rgba(0, 0, 0, 0.2);
+     /* Typography */
+     font-size: 17px;
+     line-height: 40px;
+     color: white;
+     text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
+     text-align: right;
+    letter-spacing: 1px;
+  }
+
+  .button {
+    cursor: pointer;
+    text-align: center;
+    font-size: 20px; 
+    background: #f1ff92;
+    box-shadow: 0px 4px #9da853;
```

Comment:
The overall styling of the app doesn't match the project specifications. please update.